index.js

```diff
@@ -9,16 +9,6 @@ module.exports = {
   },
 
   included() {
```

Comment:
The whole method can be removed I think?

Reply:
I've kept it, because last time something was added to included we forgot to call super. but you're right it can go.